xbmc/video/videosync/VideoSyncCocoa.cpp

```diff
@@ -33,6 +33,28 @@
 #if defined(TARGET_DARWIN_OSX)
 #include <QuartzCore/CVDisplayLink.h>
 #include "osx/CocoaInterface.h"
+#endif
+
+void CVideoSyncCocoa::OnLostDevice()
+{
+  if (!m_displayLost)
+  {
+    m_displayLost = true;
+    m_lostEvent.Wait();
```

Comment:
Ah I see.  Sorry, trying to learn as I go here and get attention to this problem at the same time.

So on OSX, it shouldn't have to deal with OnLostDevice at all?